hw2/index.js

```diff
@@ -0,0 +1,44 @@
+function NamedOne (firstName, lastName) {
+
+    // this.lastName = lastName;
```

Comment:
@matveyprupas Please, remove the comments

Reply:
Done